dll/win32/psapi/main.c

```diff
@@ -0,0 +1,76 @@
+/*
+ * PROJECT:     ReactOS Kernel
+ * LICENSE:     MIT (https://spdx.org/licenses/MIT)
+ * PURPOSE:     PSAPI Win2k3 style entrypoint
+ * COPYRIGHT:   Copyright 2025 Justin Miller <justin.miller@reactos.org>
```

Comment:
Technically this code you've moved there is [(C) 2013 Pierre Schweitzer, commit c01603fd5 (r60797)](https://github.com/reactos/reactos/commit/c01603fd519fde8204848bb36d109bf82cfb873b)